test_input_templates/module04b/Module04b.test.json.tmpl

```diff
@@ -8,10 +8,7 @@
 
   "Module04b.cohort": {{ test_batch.batch_name | tojson }},
   "Module04b.contig_list": {{ reference_resources.primary_contigs_list | tojson }},
-  "Module04b.regeno_coverage_medians": [
-    "gs://broad-dsde-methods-eph/Module04StressTest/Module04/6d90e729-0cd1-4752-94b3-e0ad128c6dc4/call-GenotypeDepthPart2/GenotypeDepthPart2/2b962a21-897e-49ac-8055-118ba459f7e4/call-RDTestGenotypeOver5kb/shard-0/cacheCopy/gt5kb.aaaaaa.bed.median_geno",
-    "gs://broad-dsde-methods-eph/Module04StressTest/Module04/6d90e729-0cd1-4752-94b3-e0ad128c6dc4/call-GenotypeDepthPart2/GenotypeDepthPart2/2b962a21-897e-49ac-8055-118ba459f7e4/call-RDTestGenotypeOver5kb/shard-1/cacheCopy/gt5kb.aaaaab.bed.median_geno"
-  ],
+  "Module04b.regeno_coverage_medians": ["gs://broad-dsde-methods-eph/Module04StressTest/Module04/2430c16a-23ee-44b4-9305-b35bcba352d9/call-GenotypeDepthPart2/GenotypeDepthPart2/f1483c4e-9e57-47ff-926f-3b354a2aa678/call-MergeRegenoCoverageMedians/test_large.regeno.coverage_medians_merged.bed"],
```

Comment:
Let's take this opportunity to fix this template. Can you redefine `regeno_coverage_medians`, `RD_depth_sepcutoffs`, and `depth_vcfs` in the `test_batch`, i.e. in `input_values/test_batch_large.json`? Also let's move these inputs to `gs://gatk-sv-resources/test/module04b/input/`.

Reply:
Done - reformatted template, moved regeno_coverage_medians, and it turns out the other files were already in that bucket under 04 outputs. Tested template generation